### PR TITLE
origin_info mapping specs - turn some back on, reversing PR #2130

### PR DESCRIPTION
**Pull request by ndushay**
NOTE:  do not squash commits!!!!!

## Why was this change made?

After #2173 (beefing up originInfo mapping specs to ensure roundtrip mapping stability), it seemed prudent to turn on as many originInfo specs as possible to trap for errors.   Thus

- this PR is a partial reversion of PR #2130.   Please keep commits separate!!!  (do not squash)

The reversal didn't need to be 100%, and I was able to increase passing origin_info specs by doing a bit more normalization of cocina created from MODS and MODS created from cocina - fewer empty elements (yay!)

### this branch

```
Status (n=10000):
  Success:   9829 (98.29%)
  Different: 103 (1.03%)
  To Cocina error:     0 (0.0%)
  To Fedora error:     0 (0.0%)
  Missing (no descMetadata):     68 (0.68%)
```

only in this branch (when keeping date fields with any attributes): 
- dateIssued with encoding and type and no value pops up in roundtrip xml (ends up in cocina) - vj932ns8042

only in this branch (when removing all date fields with no value)
- qx562pf7510
- xv158sd4671

- nc238hg8745 gets diff originInfo b/c dateOther with a type attr was removed ...

diffs in roundtrip xml or cocina between branches:

- fg349td3577, jk090bg7477 , kq586bq1216, nc238hg8745- this branch removed empty dateCreated under event with only type

- dateOther with type manufacturer gets no type in round trip;  this branch removed empty value in cocina  but no xml diff between branches - jz402xk5530

### main branch

```
Status (n=10000):
  Success:   9831 (98.31%)
  Different: 101 (1.01%)
  To Cocina error:     0 (0.0%)
  To Fedora error:     0 (0.0%)
  Missing (no descMetadata):     68 (0.68%)
```

### CODE REVIEW

**Comment by ndushay** [comment]
@arcadiafalcone 

Does this MODS snippet have any value?

```
 <originInfo eventType="production">
    <dateCreated/>
  </originInfo>
```

Is it okay if we normalize it out?

File: spec/services/cocina/mapping/descriptive/mods/origin_info_spec.rb [hunk 1]
```diff
@@ -31,172 +31,179 @@
...
-    xit 'to be implemented: updated spec'
-
-    let(:mods) do
-      <<~XML
-        <originInfo eventType="publication">
-          <copyrightDate>1980</copyrightDate>
-        </originInfo>
-      XML
-    end
+    it_behaves_like 'MODS cocina mapping' do
```
Comment (ndushay) [inline spec/services/cocina/mapping/descriptive/mods/origin_info_spec.rb:34]: reverted mapping changes to get more originInfo mappings exercised

File: spec/services/cocina/mapping/descriptive/mods/origin_info_spec.rb [hunk 2]
```diff
@@ -31,172 +31,179 @@
...
-           </place>
-           <publisher>Izdatelʹstvo "Vesʹ Mir"</publisher>
-           <dateIssued>2019</dateIssued>
-        </originInfo>
-        <originInfo eventType="copyright notice">
-           <copyrightDate>©2018</copyrightDate>
-        </originInfo>
-      XML
-    end
+    it_behaves_like 'MODS cocina mapping' do
```
Comment (ndushay) [inline spec/services/cocina/mapping/descriptive/mods/origin_info_spec.rb:69]: reverted mapping changes to get more originInfo mappings exercised

File: spec/services/cocina/mapping/descriptive/mods/origin_info_spec.rb [hunk 3]
```diff
@@ -330,56 +337,56 @@
...
-        </originInfo>
-        <originInfo eventType="publication">
-          <dateIssued>1901</dateIssued>
-          <place>
-            <placeTerm type="text">London</placeTerm>
-          </place>
-        </originInfo>
-      XML
-    end
+    it_behaves_like 'MODS cocina mapping' do
```
Comment (ndushay) [inline spec/services/cocina/mapping/descriptive/mods/origin_info_spec.rb:340]: reverted mapping changes to get more originInfo mappings exercised

File: spec/services/cocina/mapping/descriptive/mods/origin_info_spec.rb [hunk 4]
```diff
@@ -571,142 +578,68 @@
...
-    let(:mods) do
-      <<~XML
-        <originInfo displayLabel="Origin" eventType="production">
-          <place>
-            <placeTerm type="text">Stanford (Calif.)</placeTerm>
-          </place>
-        </originInfo>
-      XML
-    end
+    it_behaves_like 'MODS cocina mapping' do
```
Comment (ndushay) [inline spec/services/cocina/mapping/descriptive/mods/origin_info_spec.rb:581]: reverted mapping changes to get more originInfo mappings exercised

File: spec/services/cocina/mapping/descriptive/mods/origin_info_spec.rb [hunk 5]
```diff
@@ -571,142 +578,68 @@
...
-              }
-            ]
-          }
-        ]
-      }
-    end
-  end
-
-  describe 'Multilingual edition' do
     it_behaves_like 'MODS cocina mapping' do
```
Comment (ndushay) [inline spec/services/cocina/mapping/descriptive/mods/origin_info_spec.rb:611]: reverted mapping changes to get more originInfo mappings exercised

File: spec/services/cocina/mapping/descriptive/mods/origin_info_spec.rb [hunk 6]
```diff
@@ -760,873 +705,939 @@
...
-           <frequency>Irregular</frequency>
-           <place>
-              <placeTerm type="text">[Ruijin] in Chinese</placeTerm>
-           </place>
-           <publisher>Zhong yang ge ming jun shi wei yuan hui zong wei sheng bu in Chinese</publisher>
-        </originInfo>
-      XML
-    end
+  describe 'Multiple originInfo elements with and without eventTypes' do
+    it_behaves_like 'MODS cocina mapping' do
```
Comment (ndushay) [inline spec/services/cocina/mapping/descriptive/mods/origin_info_spec.rb:1226]: reverted mapping changes to get more originInfo mappings exercised

**Comment by arcadiafalcone** [comment]
OK to drop originInfo with no content and eventType attribute.

**Review by arcadiafalcone — APPROVED** [review]

**Comment by ndushay** [comment]
this PR has been superseded.

**Pull request closed without merging**